Similar to Arch Linux and Ubuntu, Alpine compiles SystemC with C++17
by default. Thus also include it in the lsb_release check.

diff -upr verilator-5.004.orig/configure.ac verilator-5.004/configure.ac
--- verilator-5.004.orig/configure.ac	2023-01-05 18:37:09.327766698 +0100
+++ verilator-5.004/configure.ac	2023-01-05 18:37:19.997804512 +0100
@@ -355,7 +355,7 @@ AC_SUBST(CFG_CXXFLAGS_PROFILE)
 #_MY_CXX_CHECK_SET(CFG_CXXFLAGS_STD_NEWEST,-std=gnu++20)
 #_MY_CXX_CHECK_SET(CFG_CXXFLAGS_STD_NEWEST,-std=c++20)
 case "$(which lsb_release 2>&1 > /dev/null && lsb_release -d)" in
-*Arch*Linux* | *Ubuntu*22.04*)
+*Alpine*Linux* | *Arch*Linux* | *Ubuntu*22.04*)
 _MY_CXX_CHECK_SET(CFG_CXXFLAGS_STD_NEWEST,-std=gnu++17)
 _MY_CXX_CHECK_SET(CFG_CXXFLAGS_STD_NEWEST,-std=c++17)
 ;;
